test/e2e/rte/metrics.go

```diff
@@ -36,7 +36,7 @@ import (
 	e2etestenv "github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/utils/testenv"
 )
 
-var _ = ginkgo.Describe("[RTE] metrics", func() {
+var _ = ginkgo.Describe("[RTE][Monitoring]", func() {
```

Comment:
ok, now the name is all labels. Which works, but feels weird. It should be something like
```
Describe("[RTE][Monitoring] metrics", func() {})
```